examples/sandbox/src/pages/ecommerce/Provider.jsx

```diff
@@ -0,0 +1,5 @@
+import { SearchProvider } from "@elastic/react-search-ui";
+
+export const Provider = ({ config, children }) => {
+  return <SearchProvider config={config}>{children}</SearchProvider>;
+};
```

Comment:
I suppose we don't need that component. Can we do the same in a parent component?

Reply:
added this to avoid repetition to use `SearchProvider` in other parts of code in Category, Search, Navigation & productCarousel.

Reply:
updated in https://github.com/elastic/search-ui/pull/1130/commits/a1d15d2242e4925b4112970b8b3c3d9408d4cd46